components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -0,0 +1,259 @@
+
+## Installation
+
+[Chef Automate](https://www.chef.io/automate/) provides a full suite of enterprise capabilities for workflow, node visibility, and compliance. The integration between a Chef Automate server and a ServiceNow instance requires the following components:
+
+- Chef Automate Scoped Application
+- Chef Automate Server
+
+The installation personnel performing the task of the Chef Automate Integration App in the ServiceNow ecosystem must have basic understanding of the ServiceNow system.
+
+### System Requirements
+
+The ServiceNow instance must be reachable on port 443 from Chef Automate.
+
+### Software Requirements
+
+- A running [Chef Automate](https://docs.chef.io/chef_automate.html) instance.
+- A running [ServiceNow](https://www.servicenow.com/) instance
+
+### Required ServiceNow Plugins
+
+Install following ServiceNow plugins:
+
+- System Import Sets `com.glide.system_import_set`, min version 1.0.0.
+- Configuration Management (CMDB) 1.1.
+- Configuration Management for Scoped Apps (com.snc.cmdb.scoped) 1.0.0.
+
+Locate the **System Import Sets**, **CMDB 1.1**, and **Configuration Management for Scoped App** plugins by navigating to the **System Applications** > **All Available Applications** > **All** section in the ServiceNow application.
+
+{{< figure src="/images/automate/snow_integration_plugins.png" alt="Plugins">}}
+
+### Install the Chef Automate Integration App
+
+The Chef Automate Integration App exposes the REST API endpoint for communication between Chef Automate and the ServiceNow instance. Chef Automate sends HTTPS JSON data feeds to the Chef Automate Integration App in a ServiceNow instance to create and update the _ServiceNow CMDB_ tables, client run, and compliance report tables.
+
+1. Navigate to the ServiceNow store at <https://store.servicenow.com>.
+2. Search for **Chef Automate**.
+3. Select the **Chef Automate Integration App**.
+4. Select **Get** and follow the instructions by specifying your ServiceNow credentials.
+5. Open your **ServiceNow Service Management** application
+6. Select **System Applications** > **All Available Applications** > **All** menu.
+7. Find the application using the filter criteria and search bar.
+   You can search for the application by its `Name` or `ID`. If you cannot find an application, you may have to request it from the ServiceNow Store.
+8. Select **Install**.
+
+### Assign User Roles
+
+The ServiceNow users must have the `x_chef_automate.api` role to set up the ServiceNow instance to receive data from Chef Automate. Changing the **Application Properties** defaults requires the `x_chef_automate.admin` role.
+
+The Chef Automate Integration App has three default roles:
+
+- [Role x_chef_automate.admin]({{< relref "#role_x_chef_automate.admin" >}}))
+- [Role x_chef_automate.user]({{< relref "#role_x_chef_automate.user" >}})
+- [Role x_chef_automate.api]({{< relref "#role_x_chef_automate.api" >}})
```

Comment:
These links don't go anywhere. I think this will work.

```suggestion
- [Role x_chef_automate.admin]({{< relref "servicenow_integration_reference/#role-x_chef_automateadmin" >}})
- [Role x_chef_automate.user]({{< relref "servicenow_integration_reference/#role-x_chef_automateuser" >}})
- [Role x_chef_automate.api]({{< relref "servicenow_integration_reference#role_x_chef_automateapi" >}})
```

Reply:
How did those even deploy?

Reply:
Hugo doesn't check headings.

Reply:
:)